### PR TITLE
[ci skip] Add code examples for Module#anonymous? documentation

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

File: activesupport/lib/active_support/core_ext/module/anonymous.rb [hunk 1]
```diff
@@ -7,6 +7,13 @@ class Module
   #   m = Module.new
   #   m.name # => nil
   #
+  # +anonymous?+ method return +true+ if module not have a name:
```
Comment (rafaelfranca) [inline activesupport/lib/active_support/core_ext/module/anonymous.rb]: `true` instead of `+true+`

**Comment by davydovanton** [comment]
@rafaelfranca fixed

File: activesupport/lib/active_support/core_ext/module/anonymous.rb [hunk 2]
```diff
@@ -7,6 +7,13 @@ class Module
   #   m = Module.new
   #   m.name # => nil
   #
+  # +anonymous?+ method return true if module not have a name:
```
Comment (dskecse) [inline activesupport/lib/active_support/core_ext/module/anonymous.rb]: ... returns true if module does not have...

Comment (davydovanton) [inline activesupport/lib/active_support/core_ext/module/anonymous.rb]: oops, sorry, fixed!

**Comment by davydovanton** [comment]
@rafaelfranca thanks! :green_heart:

**Pull request merged**